readme.md

```diff
@@ -1 +1,18 @@
 ## e-motion-mobile
+
+### Local development
+
+Download necessary dependencies (packages):
+```shell
+flutter pub get
+```
+
+Check flutter installation:
```

Comment:
I think that `flutter doctor` should be first and after that `flutter pub get`

Reply:
Good idea